packages/sdk/src/lib/types/coreTypes.ts

```diff
@@ -239,6 +239,18 @@ export type TransferResponse = {
 };
 
 // ---------------------ProcessProtectedData Types------------------------------------
+export type ProcessProtectedDataStatuses =
+  | 'FETCH_PROTECTED_DATA_ORDERBOOK'
+  | 'FETCH_APP_ORDERBOOK'
+  | 'FETCH_WORKERPOOL_ORDERBOOK'
+  | 'PUSH_REQUESTER_SECRET'
+  | 'PROCESS_PROTECTED_DATA_REQUESTED'
+  | 'CONSUME_TASK_ACTIVE'
+  | 'CONSUME_TASK_ERROR'
+  | 'CONSUME_TASK_COMPLETED'
```

Comment:
We'll have only one here instead of these three: "CONSUME_TASK"

Reply:
Commit: d87b9353854ec0d7169ccfe64c595f766ad00a73